Lib/asyncio/taskgroups.py

```diff
@@ -273,3 +280,30 @@ def _on_task_done(self, task):
             self._abort()
             self._parent_cancel_requested = True
             self._parent_task.cancel()
+
+    def cancel(self):
```

Comment:
what do you think about supporting cancel messages here?

Reply:
I asked on Gitter, but I'm still unclear about how such a message would be accessed and surfaced.

Reply:
It could be logged by the task that gets cancelled, or useful in debugging

Reply:
I would keep it as-is and maybe add a message in the follow-up PR; this PR is big enough for the review.